### PR TITLE
update camera according to new c++ interface

### PR DESCRIPTION
**Pull request by JonasVautherin**
Connects to https://github.com/dronecore/DroneCore/issues/334.

### CODE REVIEW

File: camera/camera.proto [hunk 1]
```diff
@@ -93,16 +93,14 @@ message CameraStatusResponse {
     CameraStatus camera_status = 1;
 }
 
-message SubscribeCurrentSettingRequest {}
-message CurrentSettingResponse {
+message SubscribeCurrentSettingsRequest {}
+message CurrentSettingsResponse {
     Setting current_setting = 1;
```
Comment (julianoes) [inline camera/camera.proto]: This should be `repeated` too, right?

File: camera/camera.proto [hunk 2]
```diff
@@ -93,16 +93,14 @@ message CameraStatusResponse {
...
 }
 
 message SubscribePossibleSettingsRequest {}
 message PossibleSettingsResponse {
-    string setting_id = 1;
-    string setting_description = 2;
-    repeated Option option = 3;
+    repeated Setting settings = 1;
```
Comment (julianoes) [inline camera/camera.proto]: This is correct, I think.

File: camera/camera.proto [hunk 3]
```diff
@@ -19,8 +19,8 @@ service CameraService {
     rpc SubscribeVideoStreamInfo(SubscribeVideoStreamInfoRequest) returns(stream VideoStreamInfoResponse) {}
     rpc SubscribeCaptureInfo(SubscribeCaptureInfoRequest) returns(stream CaptureInfoResponse) {}
     rpc SubscribeCameraStatus(SubscribeCameraStatusRequest) returns(stream CameraStatusResponse) {}
-    rpc SubscribeCurrentSetting(SubscribeCurrentSettingRequest) returns(stream CurrentSettingResponse) {}
-    rpc SubscribePossibleSettings(SubscribePossibleSettingsRequest) returns(stream PossibleSettingsResponse) {}
+    rpc SubscribeCurrentSettings(SubscribeCurrentSettingsRequest) returns(stream CurrentSettingsResponse) {}
+    rpc SubscribePossibleSettingOptions(SubscribePossibleSettingOptionsRequest) returns(stream PossibleSettingOptionsResponse) {}
```
Comment (JonasVautherin) [inline camera/camera.proto]: The C++ interface says `SubscribePossibleSettings`. I would keep it.

Comment (JonasVautherin) [reply camera/camera.proto]: My bad, I had not seen https://github.com/dronecore/DroneCore/pull/432.

**Pull request merged**